engine/crates/engine/src/context.rs

```diff
@@ -127,6 +126,12 @@ pub struct QueryEnvInner {
     pub cache_control: CacheControl,
 }
 
+impl QueryEnvInner {
+    pub fn operation_name(&self) -> &Option<String> {
+        &self.operation_metadata.name
+    }
```

Comment:
Is there a good reason to always use this fake operation_name?  

I can live with it as a hack for analytics but outside of that pretending that the operation actually had a name when it didn't seems like it could cause problems...

Reply:
hum, seems like the opposite could also be true if we're not consistent. The operation name is only necessary when deciding which operation to use in the document afaik. Don't have a strong opinion, I just wasn't a fan of the inconsistency I guess 🤷 Want me to revert that part?